Makefile

```diff
@@ -140,18 +140,18 @@ undeploy: ## Undeploy controller from the K8s cluster specified in ~/.kube/confi
 # Note that ClusterServiceVersion is not touched as is supposed to be verified manually.
 # The install strategy of ClusterServiceVersion contains the deployment which is not copied over from config/ either.
 olm-manifests: manifests
-	cp -f config/crd/bases/externaldns.olm.openshift.io_externaldnses.yaml $(BUNDLE_MANIFEST_DIR)/externaldns.olm.openshift.io_crd.yaml
-	cp -f config/rbac/role.yaml $(BUNDLE_MANIFEST_DIR)/external-dns-operator_rbac.authorization.k8s.io_v1_clusterrole.yaml
-	cp -f config/rbac/role_binding.yaml $(BUNDLE_MANIFEST_DIR)/external-dns-operator_rbac.authorization.k8s.io_v1_clusterrolebinding.yaml
-	cp -f config/rbac/auth_proxy_role.yaml $(BUNDLE_MANIFEST_DIR)/external-dns-operator-auth-proxy_rbac.authorization.k8s.io_v1_clusterrole.yaml
-	cp -f config/rbac/auth_proxy_role_binding.yaml $(BUNDLE_MANIFEST_DIR)/external-dns-operator-auth-proxy_rbac.authorization.k8s.io_v1_clusterrolebinding.yaml
-	cp -f config/rbac/auth_proxy_service.yaml $(BUNDLE_MANIFEST_DIR)/external-dns-operator-auth-proxy_v1_service.yaml
-	# opm is unable to find CRD if the standard yaml --- is at the top
-	sed -i -e '/^---$$/d' -e '/^$$/d' $(BUNDLE_MANIFEST_DIR)/*.yaml
-	# as per the recommendation of 'operator-sdk bundle validate' command
-	# doing the best (yaml regexp is far from perfect) to strip the metadata.namespace from the bundle manifests
-	# trying to not touch cluster level resources as they don't have the namespace
-	for f in $$(\grep -l 'kind: *\(Service\|ConfigMap\|Secret\|Role\) *$$' $(BUNDLE_MANIFEST_DIR)/*.yaml); do sed -i '/namespace:/d' $${f};done
+	@cp -f config/crd/bases/externaldns.olm.openshift.io_externaldnses.yaml $(BUNDLE_MANIFEST_DIR)/externaldns.olm.openshift.io_crd.yaml
```

Comment:
I just found out about the `.SILENT` [fake target](https://www.gnu.org/software/make/manual/html_node/Special-Targets.html#Special-Targets) which I think is what you want to use here.